draft-poidt-ccamp-actn-poi-pluggable.md

```diff
@@ -237,29 +264,184 @@ Px = DWDM (coherent pluggable) Router ports
-   physical component must be shared on MPI.  The above statement is
-   assumed as the Domain PNC (e.g.  O-PNC) may not be able to get
-   information from or set parameters to a node belonging to a different
-   domain (e.g.  P-PNC).
-
-   The reason of this change is due to the following statements:
-
-   O-PNC routing and wavelength assignment
-
-   > The MDSC can ask the O-PNC to set an optical circuit between two
+   responsible to coordinate the operations across the different 
+   domains having the visibility of the whole network multi-domain and
+   multi-layer network topology.
+
+   An architecture analysis has already been carried out by the MANTRA 
+   sub-group in the OOPT – (Open Optical & Packet Transport)a Telecom
+   Infra Project (TIP) group Project Group in the MANTRA whitepaper 
+   {{MANTRA-whitepaper-IPoWDM}}.
+
+   Two different architectural options have been identified, namely:
+~~~~ ascii-art
+     - Option 1: Dual SBI management of IPoDWDM routers
+     - Option 2: Single SBI management of IPoDWDM routers
+~~~~
+
+   Both the options foresee that the packet and the optical domain are 
+   kept separate from a management perspective.
+
+
+## Option 1 - Dual SBI management of IPoWDM routers
+
+The figure {{my_figure-2}} describes the architecture of option 1.
+
+
+~~~~ ascii-art
+
+                        +----------+
+                        |   MDSC   |
+                        +--+----+--+
+                           |    |  
+  MPI interface    +-------+    +-------+
+                   |                    | 
+              +----+----+          +----+----+
+              |  P-PNC  |          |  O-PNC  |
+              +--+---+--+          +----+----+
+                 ^   ^               ^  ^  ^
+                 |   |               |  |  |
+         +-------+   +---------------|--|--|------+
+         |                           |  |  |      |  
+         |   +-----------------------+  |  +--+   |
+         |   |               +----------+     |   |
+         |   |               |                |   |
+         |   |               v                |   |
+         v   |        +--------------+        |   v
+        +-----+      /                \      +-----+
+       / P.N.1 \..../..O.N--------O.N..\..../ P.N.2 \
+       \       /    \  Optical Domain  /    \       / 
+        +-----+      \                /      +-----+
+                      +--------------+
+
+   P.N. = Packet/Optical Node (IPoWDM router) 
+   O.N. = Optical Switching DWDM Node (ROADM)
+   ROADM = Lambda/Spectrum switch
+~~~~
+{: #my_figure-2 title="Dual SBI management Scenario"}
+
+
+   The peculiarity of this option consists of the fact that both the 
+   packet SDN controller (P-PNC) and the optical SDN controller (O-PNC)
+   have access to the coherent pluggable optics on the routers. 
+   The P-PNC is the only entity allowed to configure them, while the
+   O-PNC is granted with read-only permissions to avoid database 
```

Comment:
Database inconsistencies should not occur if the appropriate responsibility demarcation is drawn and config locking performed.

There are examples of successful deployment of devices with several separate controllers, indeed some are required by particular operational practices.

Reply:
v02

Reply:
I would like to discuss this.